src/Polly/CircuitBreaker/RollingHealthMetrics.cs

```diff
@@ -24,14 +25,14 @@ public void IncrementSuccess_NeedsLock()
     {
         ActualiseCurrentMetric_NeedsLock();
 
-        _currentWindow.Successes++;
+        _currentWindow!.Successes++;
```

Comment:
Is there an annotation we can add to hint the compiler that calls to a certain method will result in this being non-null?